apps/core/lib/mix/tasks/core_db_task.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Mix.Tasks.Core.Resetdb do
```

Comment:
We already have a handy [alias](https://github.com/aviabird/snitch/blob/feature%2Fcustom_mix_task/apps/core/mix.exs#L69) for this.

Reply:
Also rename this file to maybe `seed.ex`

Reply:
I think we should keep it the same keeping in mind that it will be required in future when our app core is used as a library and we will have to use this task we won't be using alias then.

Reply:
Hmmm... that's a valid point. Then we better rename this to `snitch.setup` or snitch.db.setup` And seed should be renamed to `snitch.seed`.
I setup a project with snitch as dependency and `core.resetdb` and `core.seed` could not be found.